tutorials/asr/08_ASR_with_Subword_Tokenization.ipynb

```diff
@@ -612,7 +612,7 @@
         "\r\n",
         "We will use a Citrinet model to demonstrate the usage of subword tokenization models for training and inference. Citrinet is a [QuartzNet-like architecture](https://arxiv.org/abs/1910.10261), but it uses subword-tokenization along with 8x subsampling and [Squeeze-and-Excitation](https://arxiv.org/abs/1709.01507) to achieve strong accuracy in transcriptions while still using non-autoregressive decoding for efficient inference.\r\n",
         "\r\n",
-        "We'll be using the **Neural Modules (NeMo) toolkit** for this part, so if you haven't already, you should download and install NeMo and its dependencies. To do so, just follow the directions on the [GitHub page](https://github.com/NVIDIA/NeMo), or in the [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/v1.0.0b1/).\r\n",
+        "We'll be using the **Neural Modules (NeMo) toolkit** for this part, so if you haven't already, you should download and install NeMo and its dependencies. To do so, just follow the directions on the [GitHub page](https://github.com/NVIDIA/NeMo), or in the [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/index.html/).\r\n",
```

Comment:
Should give refs to the stable or the current version of that file?
So that when user changes the version, they see the corresponding pages.

Reply:
Current version keeps changing, so link will need to constantly be updated. Stable always exists

Reply:
Isn't there a way to give ref to the same version as the current file? There may be way for that if they support versioning.

Reply:
Inside URL links, bash / python variable resolution doesnt work, so not possible.